redash/models.py

```diff
@@ -570,10 +569,9 @@ def resume(self):
     def add_group(self, group, view_only=False):
         dsg = DataSourceGroup(group=group, data_source=self, view_only=view_only)
         db.session.add(dsg)
+        #setattr(self, 'data_source_groups', dsg)
```

Comment:
Should this line just be removed then? It wasn't be used prior and not now either.